mrp_unbuild_cust_alu_analytics/views/mrp_unbuild_process_type_views.xml

```diff
@@ -11,6 +11,7 @@
                             <group>
                                 <field name="name"/>
                                 <field name="cmplanta_id"/>
+                                <field name="company_id" invisible="True"/>
```

Comment:
If company becomes invisible, we could never select it when requiered (e.g. in a multi-company scenario). It's not our current situation (customer doesn't work with multi-company), but is not right. Try this instead (https://github.com/odoo/odoo/blob/4cc086d330b73514fbc64a7fcf22d8a7a9f1b691/addons/mrp/views/mrp_bom_views.xml#L70C29-L70C119):
```suggestion
                                <field name="company_id" groups="base.group_multi_company" options="{'no_create': True}"/>
```

User won't see company field if it's working in a single-company scenario, but in a hypothetical future scenario with two or more companies it will be shown.

Reply:
Done